src/adapters/EOFeedAdapter.sol

```diff
@@ -91,7 +97,7 @@ contract EOFeedAdapter is IEOFeedAdapter, Initializable {
         IEOFeedManager.PriceFeed memory priceData = _feedManager.getLatestPriceFeed(_feedId);
         return (
             uint80(priceData.eoracleBlockNumber),
-            int256(priceData.value),
+            _normalizePrice(priceData.value),
```

Comment:
missed normalization in getRoundData
```
    function getRoundData(uint80) external view returns (uint80, int256, uint256, uint256, uint80) {
        IEOFeedManager.PriceFeed memory priceData = _feedManager.getLatestPriceFeed(_feedId);
        return (
            uint80(priceData.eoracleBlockNumber),
            int256(priceData.value),
```

Reply:
@ArielLaub we should add a test for the case of different decimals - so we catch these cases as well